docs/operator.rst

```diff
@@ -2052,6 +2052,13 @@ and emits the resulting collection as a single item. For example::
     [1,2,3,4]
     Done
 
+
+The difference between ``toList`` and ``collect`` is that while ``collect`` produces
```

Comment:
Let's put this in a note admonition. Also, I believe `collect` automatically flattens the input (check the collect section to confirm) so we should mention that here as well.

Reply:
It doesn't flatten the channel, but the outputs are indeed different.

```
Channel.of([1, [2, 3]]).collect().view { "collect: $it" }
Channel.of([1, [2, 3]]).toList().view { "toList: $it" }
```
The output is:
```
toList: [[1, [2, 3]]]
collect: [1, [2, 3]]
```

Reply:
This is because of how you are initializing the channel with a single list item, you should either use `fromList` or remove the outer list brackets:
```
> Channel.of([1, [2, 3]]).collect().view { "collect: $it" }  
collect: [1, [2, 3]]

> Channel.fromList([1, [2, 3]]).collect().view { "collect: $it" }  
collect: [1, 2, 3]

> Channel.of(1, [2, 3]).collect().view { "collect: $it" }  
collect: [1, 2, 3]

# collect only flattens one level
> Channel.of(1, [2, [3]]).collect().view { "collect: $it" } 
collect: [1, 2, [3]]
```